linux/device.go

```diff
@@ -1081,11 +1081,14 @@ func RescanForCapacityUpdates(devicePath string) error {
 		// multipathd takes either mpathx(without prefix) or /dev/dm-x as input
 		devicePath = strings.TrimPrefix(devicePath, "/dev/mapper/")
 		// reload multipath map to apply new size
-		args = []string{"reload", "map", devicePath}
-		_, _, err = util.ExecCommandOutput("multipathd", args)
+		args = []string{"resize", "map", devicePath}
```

Comment:
Didn't we attempt resize 5 times prior to reaching here? 
if multipathd was timing out before, any reason it wouldn't timeout now. Is the fix to just attempt another try?

Reply:
Yes, exactly. We need to let NodeExpand to retry as if multipathd keeps timing out we cannot update the device size.